policyengine_us/parameters/gov/states/al/tax/income/deductions/itemized/sources.yaml

```diff
@@ -2,18 +2,21 @@ description: Alabama allows for the following itemized deductions.
 values:
   2021-01-01:
     - real_estate_taxes # Section (a), (3), (d)
+    - employee_payroll_tax # Section (a), (3), (a)
+    - self_employment_tax # Section (a), (3), (a)
     # Legal code specifies charitable contributions to the extent deductible in federal law.
     # Tax form allows deducting all contributions, not capped by AGI as in federal law.
     # We adopt the legal code definition.
     - charitable_deduction # Section (a), (10)
-    # Redundant with taxsim_tfica, but broken out to keep in PolicyEngine variables.
     - al_medical_expense_deduction # Section (a), (13)
+    - al_casualty_loss_deduction # Section (a), (6)
     - al_misc_deduction # Section (a), (23)
-    - al_mortgage_interest_deduction # Section (a), (2)
-    - al_investment_interest_deduction # Section (a), (2)
+    - mortgage_interest # Section (a), (2)
```

Comment:
@MaxGhenis 
Do we have a variable for just railroad retirement taxes (tier 1)
I know we use tax_unit_taxable_social_security but I am not sure if we can use that for exclusively railroad ret. benefits

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/7daf302a-1985-43b9-9b9a-73a6522b15b8)